Steps/Steps.module.less

```diff
@@ -16,18 +16,17 @@
 			font-weight: @sand-steps-step-number-font-weight;
 		}
 
-		&.dots {
-			font-size: @sand-steps-step-dot-font-size;
+		&.highlightCurrentOnly {
+			font-size: @sand-steps-step-highlight-current-only-font-size;
 			opacity: @sand-steps-step-past-opacity;
```

Comment:
I don't think we need this opacity here.

Reply:
It's removed.